src/main/java/org/folio/services/group/GroupFiscalYearTotalsService.java

```diff
@@ -220,44 +224,45 @@ private GroupFiscalYearSummary buildGroupFiscalYearSummary(String fiscalYearId,
                                                                        GroupFiscalYearSummaryCollection groupFiscalYearSummaryCollection,
                                                                        GroupFundFiscalYearCollection groupFundFiscalYearCollection) {
+    log.debug("buildHolderSkeletons:: Building holder skeletons");
     List<GroupFiscalYearTransactionsHolder> holders = new ArrayList<>();
-    if (groupFiscalYearSummaryCollection != null
-      && groupFundFiscalYearCollection != null
-      && (!CollectionUtils.isEmpty(groupFiscalYearSummaryCollection.getGroupFiscalYearSummaries()))
-      && (!CollectionUtils.isEmpty(groupFundFiscalYearCollection.getGroupFundFiscalYears())))
+    if (groupFiscalYearSummaryCollection != null && groupFundFiscalYearCollection != null
+      && (CollectionUtils.isNotEmpty(groupFiscalYearSummaryCollection.getGroupFiscalYearSummaries()))
+      && (CollectionUtils.isNotEmpty(groupFundFiscalYearCollection.getGroupFundFiscalYears())))
     {
       groupFiscalYearSummaryCollection.getGroupFiscalYearSummaries().forEach(groupFiscalYearSummary -> {
         String fiscalYearId = groupFiscalYearSummary.getFiscalYearId();
         String groupId = groupFiscalYearSummary.getGroupId();
         List<String> groupFundIds = groupFundFiscalYearCollection.getGroupFundFiscalYears().stream()
           .filter(groupFundFiscalYear -> groupFundFiscalYear.getGroupId().equals(groupId)
-                                         && groupFundFiscalYear.getFiscalYearId().equals(fiscalYearId))
+            && groupFundFiscalYear.getFiscalYearId().equals(fiscalYearId))
           .map(GroupFundFiscalYear::getFundId)
           .filter(fundId -> isBudgetExists(fundIdFiscalYearIdBudgetsMap, fundId, fiscalYearId))
           .collect(Collectors.toList());
+        log.info("buildHolderSkeletons:: Adding groupFiscalYearTransactionHolder  to holders with '{}' groupFundId(s)", groupFundIds.size());
         holders.add(new GroupFiscalYearTransactionsHolder(groupFiscalYearSummary).withGroupFundIds(groupFundIds));
       });
     }
     return holders;
   }
 
   private Future<Void> updateHoldersWithAllocations(List<GroupFiscalYearTransactionsHolder> holders,
-                                                               RequestContext requestContext) {
+                                                    RequestContext requestContext) {
     List<Future<GroupFiscalYearTransactionsHolder>> futures = new ArrayList<>();
     holders.forEach(holder ->
-       futures.add(updateHolderWithAllocations(requestContext, holder))
+      futures.add(updateHolderWithAllocations(requestContext, holder))
     );
     return collectResultsOnSuccess(futures)
-      .onSuccess(result -> log.debug("Number of holders updated with allocations: {}" , result.size()))
+      .onSuccess(result -> log.info("updateHoldersWithAllocations:: Number of holders updated with allocations: {}", result.size()))
```

Comment:
it's not conditional logic, as for me its debug level

Reply:
changed, thanks